### ISSUE TITLE
Get multiple series y-axis values from x-axis position

### ISSUE THREAD

**Issue by towsie84** [amcharts/amcharts4#1732]
I have a vertically stacked multiple y-axis chart that uses a dateAxis as the x-axis and a valueAxis for each y-axis.

Is it possible that when I click or double click at any position on the chart it will be able to get the y-axis value for each series added to the chart?

Thanks

**Comment by towsie84** [comment]
I had this working on anohter chart that used a categosrAxis as the x-axis and by tracking the cursor position as below.

`var cursorPosition = null
chart.cursor.events.on("cursorpositionchanged", function (ev) {
var xAxis = ev.target.chart.yAxes.getIndex(0);
cursorPosition = xAxis.positionToDate(xAxis.toAxisPosition(ev.target.xPosition));
});

chart.plotContainer.events.on("doublehit", function (ev) {
scope.config.chartMarkers.push({ "time": cursorPosition });
});`

I then went through the data and got the values that matched the category.

I have tried something similar using positionToDate but it isn't working.

Thansk
